tcrudge/utils/schema.py

```diff
@@ -294,8 +294,5 @@ def _generate_array(self, array):
         self._add_items(array, 'add_object')
 
     def _generate_basic(self, val):
-        if val in JS_TYPES.keys():
-            val_type = JS_TYPES[val]
-        else:
-            val_type = JS_TYPES[type(val)]
+        val_type = JS_TYPES[val] if val in JS_TYPES.keys() else JS_TYPES[type(val)]
```

Comment:
Function `Schema._generate_basic` refactored with the following changes:

- Replace if statement with if expression ([assign-if-exp](https://docs.sourcery.ai/refactorings/assign-if-exp/))